### ISSUE TITLE
Configuration Management Sychronize improve text in opening paragraph

### ISSUE THREAD

**Issue by izmeez** [backdrop/backdrop-issues#6621]
## Description of the need

Backdrop CMS currently displays the following text:

> This form allows you to synchronize configuration between sites, such as in development, staging, and production servers. This page shows you the current differences between your current configuration and changes that have been staged for deployment.

![image](https://github.com/backdrop/backdrop-issues/assets/3144571/d04847e8-c180-4c44-b1c8-33c92b1cf4f6)

Proposed text:

> This page shows the current differences between the active and staged configurations. It allows you to synchronize configuration between sites, such as in development, staging, and production servers. 

![image](https://github.com/backdrop/backdrop-issues/assets/3144571/464b57c4-30b2-4645-bd70-dae25bd14365)

**Comment by stpaultim** [comment]
I added screenshots to the original post by @izmeez. 

This seems like a pretty minor change that might be helpful. It seems like the point of the change is put more empahsis on the `diff` functionality of what is happening here. 

I haven't used this page in a while, so I'd like to see what it looks like with some actual diff's visible. I'll try to test that later.

**Comment by klonos** [comment]
> ... It seems like the point of the change is put more empahsis on the `diff` functionality of what is happening here.

Yeah, and although when I first read this I liked the proposal, I'm not so sure (but could be convinced). My reason of hesitation is the fact that the primary purpose of the page and the title of the tab is "Synchronize". So it feels right that the first sentence was the one with the "allows you to synchronize" bit.

While we're at it, "shows the current differences" is not 100% accurate - if we want to be more precise, what this page does is to list individual config changes, and provides a way to view the differences between active and staged for each changed config in a side-by-side, color-coded visual comparison. But all that is too wordy. Since this page is meant for use by more technical people, with more in-depth knowledge of development terminology (staging/production, version control etc.), perhaps we can reduce that by using "visual diff". Something like this: `This page lists any configuration changes between what is currently active and what has been staged for deployment, and allows inspecting these changes in detail via a visual diff.`

The other, "allows you to synchronize" sentence seems fine to me as is.

**Comment by klonos** [comment]
> In many ways the word `Synchronize` is a misnomer. Usually we think of synchronize as a precise action, whereas currently it is a brute force: all or none. But, it is the right word for what is needed and possible.

I guess it is what has been historically used (in Drupal as well when CMI was first implemented). If by "brute force" you mean the inability to select only specific config to be imported, then yes, that is true; however, in order to have "selective config sync" we need to have some sort of configuration dependencies, which is really difficult to implement ...otherwise, allowing arbitrary config to be imported can end up really messy (as in broken sites).

In the sense of bringing the dev/local/staging environment in par with production, then "synchronize" seems like a good choice of word to me.

Nerd-alert!: If you really want to get technical about it, the word "synchronize" is a compound of the Greek words "syn" (the equivalent of the English prefix "co-") and "chronos" (= "time"). It means to adjust two or more things so that they operate or happen at the same time. Only in computing it means "make data in two or more places identical" 🤷🏼 ...so if you really-really wanted to be 100% accurate (and still use a Greek-derived word), then you would use "homogenize" 🤓

**Comment by kiamlaluno** [comment]
Actually, *synchronize* is also used in *Synchronize watches* (*make two watches show the same time*) which gives to *synchronize* a similar meaning it has in computing. 😉

**Comment by kiamlaluno** [comment]
As for being able to choose what is synchronized, that is not mandatory. I have a smart phone application that allows me to use the same data on my smart phone on my smart watch; it does not allow to select what must be synchronized, though.
